recipes/bioconvert/meta.yaml

```diff
@@ -34,20 +41,36 @@ requirements:
     - biopython
     - mappy
     - matplotlib
+    - networkx
     - pyyaml
     - pysam
+    - pyexcel
+    - pyexcel-ods3
+    - pyexcel-xls
+    - xlrd
     - bamtools
     - bcftools
     - bedtools
     - bioawk
     - deeptools
-    - dsrc # [not osx]
+    - dsrc
+    - go ==1.10.3
```

Comment:
go is required and a specific version of it?

Reply:
go is required. We need 1.10.3 for now indeed.